test/benchmark_setup_teardown_test.cc

```diff
@@ -128,7 +128,7 @@ BENCHMARK(BM_WithRep)
     ->Repetitions(4);
 
 int main(int argc, char** argv) {
-  benchmark::Initialize(&argc, argv);
+  benchmark::Initialize(&argc, argv, benchmark::internal::helper_printer);
```

Comment:
requiring this isn't great from an API change POV.  can we default it to something (`nullptr`?) and then set it to the internal version if it's not passed in?

Reply:
This is a good refactoring to have, but I'm not a fan of the nullptr, what about the optional type? like helper_printer() = HelperPrinter?

I pushed this solution right now, but if you want the nullptr, I will be happy to revert it